tests/2017-03-01-preview/Microsoft.Insights.ActionGroup.tests.json

```diff
@@ -0,0 +1,144 @@
+{
+  "tests": [
+    {
+      "name": "Action Group Test - SMS Only",
+      "definition": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActionGroup.json#/resourceDefinitions/actiongroups",
+      "json": {
+        "type": "Microsoft.Insights/actionGroups",
+        "apiVersion": "2017-03-01-preview",
+        "location": "Global",
+        "name": "TEST Action Group - SMS only",
+        "properties": {
+          "groupShortName": "SMSOnly",
+          "enabled": true,
+          "smsReceivers": [
+            {
+              "name": "contosoSMS",
+              "countryCode": "1",
+              "phoneNumber": "5555551212"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "Action Group Test - Email Only",
+      "definition": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActionGroup.json#/resourceDefinitions/actiongroups",
+      "json": {
+        "type": "Microsoft.Insights/activityLogAlerts",
```

Comment:
This should be "Microsoft.Insights/actionGroups". Same for the other tests in this file.